dev/Common/ColorConversion.h

```diff
@@ -17,6 +17,12 @@ class Rgb
     Rgb(double r, double g, double b);
 };
 
+struct Rgba
+{
+    Rgb rgb{};
+    double a{};
+};
```

Comment:
Per other comments I think this can go away and HexToRgba can return std::tuple<Rgb, double>